src/main.js

```diff
@@ -0,0 +1,368 @@
+              <img class="event__photo" src="img/photos/1.jpg" alt="Event photo">
+              <img class="event__photo" src="img/photos/2.jpg" alt="Event photo">
+              <img class="event__photo" src="img/photos/3.jpg" alt="Event photo">
+              <img class="event__photo" src="img/photos/4.jpg" alt="Event photo">
+              <img class="event__photo" src="img/photos/5.jpg" alt="Event photo">
+            </div>
+          </div>
+        </section>
+      </section>
+    </form>`
+  );
+};
+
+// Разметка для одного дня
+const createDayTemplate = () => {
+  return (`
+    <li class="trip-days__item  day">
+      <div class="day__info">
+        <span class="day__counter">3</span>
+        <time class="day__date" datetime="2019-03-18">MAR 20</time>
+      </div>
+
+      <ul class="trip-events__list">
+        <!-- Точки маршрута -->
+      </ul>
+    </li>`
+  );
+};
+
+// Разметка точки маршрута
+const createTripEventTemplate = () => {
+  return (`
+    <li class="trip-events__item">
+      <div class="event">
+        <div class="event__type">
+          <img class="event__type-icon" width="42" height="42" src="img/icons/taxi.png" alt="Event type icon">
+        </div>
+        <h3 class="event__title">Taxi to Amsterdam</h3>
+
+        <div class="event__schedule">
+          <p class="event__time">
+            <time class="event__start-time" datetime="2019-03-18T10:30">10:30</time>
+            &mdash;
+            <time class="event__end-time" datetime="2019-03-18T11:00">11:00</time>
+          </p>
+          <p class="event__duration">30M</p>
+        </div>
+
+        <p class="event__price">
+          &euro;&nbsp;<span class="event__price-value">20</span>
+        </p>
+
+        <h4 class="visually-hidden">Offers:</h4>
+        <ul class="event__selected-offers">
+          <li class="event__offer">
+            <span class="event__offer-title">Order Uber</span>
+            &plus;
+            &euro;&nbsp;<span class="event__offer-price">20</span>
+           </li>
+        </ul>
+
+        <button class="event__rollup-btn" type="button">
+          <span class="visually-hidden">Open event</span>
+        </button>
+      </div>
+    </li>`
+  );
+};
+
+// ===========================================================
+
+const render = (container, template, place = `beforeend`) => {
+  container.insertAdjacentHTML(place, template);
+};
+
+
+const siteHeaderElement = document.querySelector(`.page-header`);
+
+const tripMainElement = siteHeaderElement.querySelector(`.trip-main`);
+render(tripMainElement, createTripInfoTemplate(), `afterbegin`);
+
+const tripInfoElement = siteHeaderElement.querySelector(`.trip-info`);
+render(tripInfoElement, createTripPriceTemplate());
+
+const tripControlsHeaders = siteHeaderElement.querySelectorAll(`.trip-controls h2`);
+render(tripControlsHeaders[0], createSiteMenuTemplate(), `afterend`);
+render(tripControlsHeaders[1], createFilterTemplate(), `afterend`);
```

Comment:
Здесь лучше не обращаться по индексу, а сразу задать имена с помощью деструктуризации
```suggestion
const [tripMenuHeader, tripFiltersHeader] = siteHeaderElement.querySelectorAll(`.trip-controls h2`);
render(tripMenuHeader, createSiteMenuTemplate(), `afterend`);
render(tripFiltersHeader, createFilterTemplate(), `afterend`);
```